### PR TITLE
Prepare sc for py transform 

### PR DESCRIPTION
**Pull request by chengxuHawkwood**

### CODE REVIEW

**Comment by chengxuHawkwood** [comment]
@yiheng

File: README.md [hunk 1]
```diff
@@ -27,10 +27,8 @@ Step-by-step Deep Leaning Tutorials on Apache Spark using [BigDL](https://github
 + [Setup env on Mac OS](https://github.com/intel-analytics/BigDL-Tutorials/blob/master/SetupMac.md) / [Setup env on Linux](https://github.com/intel-analytics/BigDL-Tutorials/blob/master/SetupLinux.md)
 
 ### Start Jupyter Server
-* Download BigDL 0.3.0([linux or mac](https://repo1.maven.org/maven2/com/intel/analytics/bigdl/dist-spark-2.2.0-scala-2.11.8-linux64/0.3.0/dist-spark-2.2.0-scala-2.11.8-linux64-0.3.0-dist.zip )) and unzip file.
-* Run ```export BIGDL_HOME=where is your unzipped bigdl folder```
-* Run ```export SPARK_HOME=where is your unpacked spark folder```
-* Run ```./start_notebook.sh```
+* Run ```pip install BigDL```
```
Comment (yiheng) [inline README.md]: pip install BigDL=0.3.0

**Comment by yiheng** [comment]
Also update the SetupLinux.md file. Some content is incorrect now. By the way, will pip install BigDL install Java?

**Review by yiheng — CHANGES_REQUESTED** [review]

**Comment by chengxuHawkwood** [comment]
@yiheng I do not think java will be installed. I will also add the steps to install java in the related files.

File: README.md (hunk 1 above)
Comment (chengxuHawkwood) [reply README.md]: I think this should be 'pip install BigDL==0.3.0' instead if 0.3.0 version is specified.

**Comment by yiheng** [comment]
LGTM

**Pull request merged**